test/shared/src/main/scala/zio/test/TestAspect.scala

```diff
@@ -100,10 +100,21 @@ object TestAspect extends TimeoutVariants {
         test <* effect.catchAllCause(cause => ZIO.fail(TestFailure.Runtime(cause)))
     }
 
+  /**
+   * Constructs an aspect that evaluates every test inside the context of a `Managed`.
+   */
+  def around[R0, E0](managed: ZManaged[R0, E0, Any]) =
```

Comment:
Since the `ZManaged` returns `Any`, what about taking `before` and `after` directly, then we can wrap them into a managed inside this function and pass them to the other `aroundTest`? Less work for user.

Reply:
I like it. Let me make that change now.